src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -867,6 +867,61 @@ LogicalResult ONNXSeluOp::inferShapes(
   return success();
 }
 
+LogicalResult ONNXSequenceInsertOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+static LogicalResult verify(ONNXSequenceInsertOp op) {
+  ONNXSequenceInsertOpAdaptor operandAdaptor = ONNXSequenceInsertOpAdaptor(op);
+
+  // These cast should be guaranteed by default verifier
+  auto seqElementType = operandAdaptor.input_sequence()
+                            .getType()
+                            .dyn_cast<mlir::onnxmlir::SeqType>()
+                            .getElementType();
+  auto elementType1 = seqElementType.dyn_cast<ShapedType>().getElementType();
+  auto insertType = operandAdaptor.tensor().getType().dyn_cast<ShapedType>();
+  auto elementType2 = insertType.getElementType();
+
+  if (elementType1 != elementType2) {
+    return op.emitError("Element types of the tensor in seqence and input "
+                        "have to be the same");
+  }
+  return success();
+}
+
+LogicalResult ONNXConcatFromSequenceOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+LogicalResult ONNXSequenceAtOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+LogicalResult ONNXSequenceConstructOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+LogicalResult ONNXSequenceEmptyOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+LogicalResult ONNXSequenceEraseOp::inferShapes(
+    std::function<void(mlir::Region &)> doShapeInference) {
+  return success();
+}
+
+// Output should an integer64
```

Comment:
typo: should an -> should be an

I do not understand the comment. The returned  type is `LogicalResult` what is the output in this context?

Reply:
Comment removed. I should add a verifier for this op.